src/main/features/claimant-response/draft/draftClaimantResponse.ts

```diff
@@ -15,6 +16,7 @@ export class DraftClaimantResponse extends DraftDocument {
   settlementAgreement?: SettlementAgreement
   alternatePaymentMethod?: PaymentIntention
   freeMediation?: FreeMediation
+  paidAmount?: PaidAmount
```

Comment:
Wrapping this with wrapper object with validation annotations will help you:
- keep CCJ stuff scoped and contained in one place
- validate task completeness because then you can do `isValid(ccjRequest)` assuming that wrapper is called `CCJRequest`.